### PR TITLE
Create package.json

### PR DESCRIPTION
**Pull request by uqureshi-21**
please merge the pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"text":"some json"}
```
Comment (umerqureshi-21) [inline package.json:1]: can this be refactored

Comment (uqureshi-21) [reply package.json:1]: yeah I have done it now

**Review by umerqureshi-21 — CHANGES_REQUESTED** [review]
I have reviewed please make the changes

**Comment by uqureshi-21** [comment]
ok the change has been made

**Comment by uqureshi-21** [comment]
ok changes have been made

**Review by umerqureshi-21 — APPROVED** [review]
ok approved this time

**Review by umerqureshi-21 — APPROVED** [review]
changes approved

**Comment by umerqureshi-21** [comment]
please close

**Pull request merged**